x-pack/plugins/siem/public/lib/helpers/index.tsx

```diff
@@ -18,3 +18,9 @@ export type WrapArrayIfExitts = <T>(value: Many<T>) => T[] | undefined;
  */
 export const asArrayIfExists: WrapArrayIfExitts = value =>
   !isUndefined(value) ? castArray(value) : undefined;
+
+export const wait = (): Promise<void> => {
+  return new Promise(resolve => {
+    return setTimeout(resolve, 10);
```

Comment:
Looks like this is used in the `first_last_seen` tests to ensure the mock data can be fetched. Is `10 seconds` necessary, and if so, that balloons our test time, correct? Looks like there's `6` usages right now, so that's another minute added to testing?

Reply:
Hahahha, it should be 0.

Reply:
sorry about that